Cargo.toml

```diff
@@ -15,7 +15,10 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kira = { version = "0.8.7", features = ["serde"] }
+bevy_math = { version = "0.13.2", features = ["mint"] }
+cpal = "0.15.3"
+kira = { version = "0.8.6", git = "https://github.com/tesselode/kira.git", branch = "main", features = ["serde"] }
+mint = "0.5.9"
```

Comment:
`mint` dep seems unused

Reply:
Ah yes, I was doing manual conversions to `mint` types but figured out how to enable mint support from `bevy_math`